### ISSUE TITLE
[Bug]: TextField border should have 3:1 minimum contrast ratio 

### ISSUE THREAD

**Issue by Himanshu-998** [adobe/spectrum-web-components#4586]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-textfield

### Expected behavior

TextField border should have 3:1 minimum contrast ratio 

### Actual behavior

TextField border does not have 3:1 minimum contrast ratio 

### Screenshots

<img width="1165" alt="Screenshot 2024-07-03 at 1 45 45 PM" src="https://github.com/adobe/spectrum-web-components/assets/39983976/6772bfe2-525a-4509-82f5-052b6fe068d6">

### What browsers are you seeing the problem in?

Chrome

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/textfield/
2. Choose System as "Express"
3. Scroll to sp-textfeild
4. Check the color contrast ratio of border or the text field

**Comment by najikahalsema** [comment]
We are not planning to make visual updates in Spectrum and will fix this in S2.
